### DISCUSSION TITLE
Twilio doesn't work anymore

### DISCUSSION THREAD

**Discussion by colossalchicken** [community/community#38836] [GitHub Education] [answered]
`https://www.twilio.com/quest/github-education` sends you to `https://www.twilio.com/try-twilio?promo=GitHubEdu`.
There is a message that claims:
> Our referral program was discontinued November 1, 2022. You can still create an account and get started with a free trial.

**Comment by radar-blip** [comment]
Worked fine for me. Maybe try again.

**Reply by colossalchicken** [reply]
no luck

**Comment by juanpflores** [comment]
Hey @colossalchicken 👋🏼 
Thanks for reporting this issue 🐛. We've pinged our team to get more details on the issue and how to solve it.

**Reply by oumjniha** [reply]
hi please i want to use my coupon code but the coupon expired any solution please

**Comment by joenash** [comment, marked as answer]
Hey folks :wave: Joe from Twilio Education here. We've got a fix coming in soon, we'll let you know when it's live, thanks for your patience!

**Reply by oumjniha** [reply]
hi please i want to use my coupon code but the coupon expired any solution please

**Reply by fabioselau077** [reply]
Hello, resolved?

**Reply by juanpflores** [reply]
Hey folks! Juan Pa here from the GitHub Education team. Unfortunately, Twilio is no longer a member of the pack.

**Comment by morganersery1** [comment]
Hi all, relaying the message on the TwlioQuest site here: https://www.twilio.com/quest

> The TwilioQuest program was impacted by the recent organizational re-structure at Twilio. As a result, TwilioQuest, Field Operators and Operator Academy will be sunset by the end of March 2023. Access to our swag store will be disabled for these programs on 03/15.
> 
> TwilioQuest will be moving to open source in the coming months, and we will be sending an update on this via the newsletter. If you’d like to continue learning with Twilio, you can join us in our [live training sessions](https://www.twilio.com/ahoy/on-demand), as well as learning on-demand with our [Youtube channel](https://www.youtube.com/watch?v=4jUMqutYmyE). If you’d like to connect with other developers in the Twilio community, join us in the [@TwilioDevs](https://discord.gg/HGwvuYNFAy) Discord Server.


We will work to see if there are any offer alternatives Twilio would like to contribute to the Student Developer Pack. Thanks for your patience while we confirm with them on this. In the meantime, I've removed the offer from the Pack so there is no longer this confusion.
